corehq/apps/commtrack/tests/util.py

```diff
@@ -185,3 +186,16 @@ def extract_balance_xml(xml_payload):
     if balance_blocks:
         return [etree.tostring(bb) for bb in balance_blocks]
     return []
+
+
+def fake_sms(user, text):
+    """
+    Don't use this with a real user
```

Comment:
the warning is helpful, though it might also be nice to explain what the intention of the function is here...